pandas/tests/base/test_value_counts.py

```diff
@@ -38,6 +38,7 @@ def test_value_counts(index_or_series_obj):
     tm.assert_series_equal(result, expected)
 
 
+@pytest.mark.filterwarnings("ignore:.*append method is deprecated.*:FutureWarning")
```

Comment:
why is this showing a warning? generally *don't* add filterwarnings in anything but the actual method testing itself (e.g. append), these can be suppressing a warning that is thrown internally which we want to fix